installer/install.iss

```diff
@@ -837,12 +838,13 @@ begin
         end
     end;
 
-    if not ExecWithCapture('"'+AppDir+'\{#MINGW_BITNESS}\bin\git.exe" config -l -z '+ExtraOptions,StdOut,StdErr,ExitCode) then begin
+    if not ExecAndCaptureOutput('"'+AppDir+'\{#MINGW_BITNESS}\bin\git.exe"', 'config -l -z '+ExtraOptions, '', SW_SHOWNORMAL, ewWaitUntilTerminated, ExitCode, Output) then begin
         if FileExists(AppDir+'\{#MINGW_BITNESS}\bin\git.exe') then
-            LogError('Unable to get system config (exit code '+IntToStr(ExitCode)+'):'+#13+#10+StdErr);
+            LogError('Unable to get system config (exit code '+IntToStr(ExitCode)+'):'+#13+#10+StringJoin(#13+#10,Output.StdErr));
     end;
 
     // Split NUL-delimited key/value pairs, extract LF that denotes end of key
+    StdOut:=StringJoin(#10,Output.StdOut);
```

Comment:
We probably want to iterate over `Output.StdOut` directly instead of joining and re-slitting this, but that was the easiest Q&D way to at least get it to compile again.

Reply:
We do that now, but there's probably at least one off-by-one error in there.